test/testing_common.py

```diff
@@ -1,12 +1,17 @@
+import asyncio
```

Comment:
See my comment above - perhaps reworking the existing test suite to be async-capable can be split out into a separate changeset that preceded lndmanage-daemon?

Reply:
I'll make a separate PR just with the preparatory async changes to unblock you with #125.

Reply:
@3nprob, I separated out the async preparation in #126, on which you could build on top of.

Reply:
> @3nprob, I separated out the async preparation in #126, on which you could build on top of.

Awesome!